services/movie.py

```diff
@@ -0,0 +1,50 @@
+from db.models import Movie, Genre, Actor
```

Comment:
Code Style: Ensure consistent use of quotes. Double quotes are preferable as per the checklist.